### PR TITLE
app-editors/ghostwriter: add missing dependency

### PR DESCRIPTION
**Pull request by stkw0**
Closes: https://bugs.gentoo.org/673318
Package-Manager: Portage-2.3.52, Repoman-2.3.12
Signed-off-by: David Roman <davidroman96@gmail.com>

### CODE REVIEW

File: app-editors/ghostwriter/ghostwriter-1.7.4.ebuild [hunk 1]
```diff
@@ -25,6 +25,7 @@ RDEPEND="
 DEPEND="${RDEPEND}
 	dev-qt/linguist-tools:5
 	dev-qt/qtconcurrent:5
+	dev-qt/qtsvg:5
```
Comment (a17r) [inline app-editors/ghostwriter/ghostwriter-1.7.4.ebuild]: ghostwriter is installing an svg icon. If this is used at runtime, you will also need to add qtsvg to `RDEPEND` even if it is not linked to.

Comment (stkw0) [reply app-editors/ghostwriter/ghostwriter-1.7.4.ebuild]: thanks, fixed.

Comment (a17r) [reply app-editors/ghostwriter/ghostwriter-1.7.4.ebuild]: thanks, but an `RDEPEND` change requires revbump.

File: app-editors/ghostwriter/ghostwriter-1.7.4-r1.ebuild [hunk 2]
```diff
@@ -7,7 +7,7 @@ inherit qmake-utils xdg-utils
 
 DESCRIPTION="Cross-platform, aesthetic, distraction-free markdown editor"
 HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
-SRC_URI="https://github.com/wereturtle/ghostwriter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/wereturtle/ghostwriter/archive/v${PV}.tar.gz -> ${PF}.tar.gz"
```
Comment (a17r) [inline app-editors/ghostwriter/ghostwriter-1.7.4-r1.ebuild]: no need to change anything here, in fact `PF` is wrong as a revbump does not change anything on our tarball, does it?

File: app-editors/ghostwriter/ghostwriter-1.7.4-r1.ebuild [hunk 3]
```diff
@@ -21,6 +21,7 @@ RDEPEND="
 	dev-qt/qtprintsupport:5
 	dev-qt/qtwebkit:5
 	dev-qt/qtwidgets:5
+	dev-qt/qtsvg:5
```
Comment (a17r) [inline app-editors/ghostwriter/ghostwriter-1.7.4-r1.ebuild]: please keep lexicographical sorting order, sorry I missed that earlier ;)

Comment (stkw0) [reply app-editors/ghostwriter/ghostwriter-1.7.4-r1.ebuild]: Thanks. Sorry for my little mistakes.

**Pull request closed without merging**